src/containers/ArticlePage/ArticlePage.jsx

```diff
@@ -123,6 +123,7 @@ class ArticlePage extends Component {
     }
 
     const { resource, topic, resourceTypes, subject, topicPath } = data;
+    const resourceTitle = topicPath.pop().name;
```

Comment:
A bit confusing calling it a resource title, when it is a topic title?

Reply:
You also need to null check. Topic path can be empty if an api call fails: https://github.com/NDLANO/ndla-frontend/blob/master/src/containers/ArticlePage/ArticlePage.jsx#L45.